### PR TITLE
Bottom Navigation Bar instead of Tab Bar

### PR DESCRIPTION
**Pull request by ManishModak**
Hi there! I came across this repository and spotted an issue that caught my attention. I'm relatively new to coding and have been working with Dart for only about 8-9 months So please let me know if there is issue with code i have written, I'm eager to learn and contribute to Hacktoberfest. Looking forward to collaborating with you all!

### CODE REVIEW

File: lib/ui/home.dart [hunk 1]
```diff
@@ -14,48 +14,53 @@ class Home extends StatelessWidget {
...
+            Navigator.pushNamed(context, StringConstants.randomRoute);
+          },
+          child: Icon(Icons.question_mark),
+        )
+            : null,
+      );
     });
   }
-}
+}
```
Comment (danger-ahead) [inline lib/ui/home.dart:66]: add a new line here.

**Comment by danger-ahead** [comment]
@ManishModak Can you share a screen recording with the changes?

File: lib/ui/home.dart (hunk 1 above)
Comment (danger-ahead) [reply lib/ui/home.dart:66]: run `dart format .` before adding the changes to git.

**Comment by danger-ahead** [comment]
@ManishModak Nice work on your part. But there was already one existing PR before yours.
Closing this as of now. If you have noticed any other improvement scopes/bugs, please feel free to open a PR.

**Comment by ManishModak** [comment]
ok no problem i will see if i can contribute in something else.

**Pull request closed without merging**